app/angular/src/client/preview/angular/app.token.ts

```diff
@@ -1,4 +1,10 @@
-import { InjectionToken } from '@angular/core';
-import { StoryFnAngularReturnType } from '../types';
+import { InjectionToken, Provider } from '@angular/core';
+import { Subject } from 'rxjs';
+import { ICollection } from '../types';
 
-export const STORY = new InjectionToken<StoryFnAngularReturnType>('story');
+export const STORY_PROPS = new InjectionToken<Subject<ICollection | undefined>>('STORY_PROPS');
```

Comment:
I am not completely against this, but I am wondering if it is worth it to loose the rest of the story data? It will break 1 or 2 of my utilities, but they aren't very important and I can probably fix them by getting the data of the selected story from Storybook's api when the observable emits. I am just trying to think if it could break or limit something important, which it may not.

Reply:
It's a private provider we shouldn't depend on it too much, I think
But we can leave the old one if we need it ? 
This one only needs props I think for the moment. but we can also use the old one (but I find it less clean ^^ )

--
For info : 
The old token type is not good ^^ this should be a `InjectionToken<Subject <StoryFnAngularReturnType>>`
--

Reply:
If needed we can keep `STORY` for now and add the more specific `STORY_PROPS`.
I just checked, `STORY` wasn't part of the public API so there should be not a lot of developers using it outside the SB contributor world.